cli/src/cast.rs

```diff
@@ -80,6 +80,11 @@ async fn main() -> eyre::Result<()> {
             };
             println!("0x{}", output);
         }
+        Subcommands::FromBin { value } => {
+            let val = unwrap_or_stdin(value)?;
+            let n: u32 = u32::from_str_radix(&val, 2).unwrap();
```

Comment:
This will only allow a single 32 bit integer to be converted, whereas `seth` converts a binary blob into hex

Reply:
@onbjerg help out a bit here. What should be the input? I imagined a String that represents binary data, aka "101010101010101..".

Reply:
Yes, but the string can be larger than 32 bits. Example input could be the bytecode of a contract in binary, which could be up to 22 kilobytes of data

Reply:
Right, awesome. Thanks for the comments, will fix it asap